package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "faros-js-client",
-  "version": "0.0.1",
```

Comment:
No need for this. Please revert

Reply:
@ted-faros we trigger releases from GitHub tags